inference-engine/tests/functional/inference_engine/cnn_network/convert_ngraph_to_cnn_network_tests.cpp

```diff
@@ -81,7 +81,7 @@ TEST(ConvertFunctionToCNNNetworkTests, OpsShouldBeConvertedToIERepresentation) {
             std::make_shared<ngraph::opset4::GRUCell>(),
             // std::make_shared<ngraph::opset4::GRUSequence>(), todo: enable after GRUSequence support
             std::make_shared<ngraph::opset4::HardSigmoid>(),
-            std::make_shared<ngraph::opset4::Interpolate>(),
+            // std::make_shared<ngraph::opset4::Interpolate>(),
```

Comment:
If this line is not needed then just remove it.

Reply:
Done.